app/javascript/components/venue/VenueEdit.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import Axios from 'axios';
+
+export default class VenueEdit extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.state = { address: '', description: '', people_capacity: '' };
```

Comment:
![image](https://user-images.githubusercontent.com/48195830/63019445-09cf3800-bea4-11e9-9efa-8cf252d185fa.png)
please, add key, I marked red in your example

Reply:
done, thanks, i've forgotten about him.